tests/unit/services/signature/gpg/GnupgSignatureVerifierTest.php

```diff
@@ -18,14 +18,14 @@
  * @covers \PharIo\Phive\GnupgSignatureVerifier
  */
 class GnupgSignatureVerifierTest extends TestCase {
-    /** @var \Gnupg|ObjectProphecy */
+    /** @var GnuPG|ObjectProphecy */
```

Comment:
This should be [`PharIo\Phive\GnuPG`](https://github.com/phar-io/phive/blob/23171fc32119c6e1cebefa4d3bbae67d25ba1e44/src/shared/GnuPG.php).